### PR TITLE
[Storage] Support new Storage Account Kind BlockBlobStorage

### PR DESCRIPTION
**Pull request by blueww**
## Description
BlockBlobStorage Kind is GA, so move it from preview to official release.
The design is part of  Azure/azure-powershell-cmdlet-review-pr#44

## Checklist

- [x] I have read the [_Submitting Changes_](https://github.com/Azure/azure-powershell/blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](https://github.com/Azure/azure-powershell/blob/master/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [x] The appropriate [change log has been updated](https://github.com/Azure/azure-powershell/blob/master/CONTRIBUTING.md#updating-the-change-log)
- [x] The PR does not introduce [breaking changes](https://github.com/Azure/azure-powershell/blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [x] If applicable, the changes made in the PR have proper test coverage
- [x] For public API changes to cmdlets:
    - [x] the changes have gone through a [cmdlet design review](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (Microsoft internal partners only)
    - [x] the cmdlet markdown files were [generated using the `platyPS` module](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/help-generation.md)

### CODE REVIEW

File: src/Storage/Storage.Management/ChangeLog.md [hunk 1]
```diff
@@ -18,6 +18,8 @@
         - Additional information about change #1
 -->
 ## Upcoming Release
+* Support Kind BlockBlobStorage when create Storage account
+       - New-AzureRmStorageAccount
```
Comment (cormacpayne) [inline src/Storage/Storage.Management/ChangeLog.md]: @blueww `New-AzStorageAccount` 😀

File: src/Storage/Storage.Management/StorageAccount/NewAzureStorageAccount.cs [hunk 2]
```diff
@@ -74,6 +74,7 @@ public class NewAzureStorageAccountCommand : StorageAccountBaseCmdlet
         [ValidateSet(AccountKind.Storage,
             AccountKind.StorageV2,
             AccountKind.BlobStorage,
+            AccountKind.BlockBlobStorage,
```
Comment (cormacpayne) [inline src/Storage/Storage.Management/StorageAccount/NewAzureStorageAccount.cs]: @blueww would you mind regenerating the help for the cmdlet so this value is listed in the `AcceptedValues` section?

Comment (blueww) [reply src/Storage/Storage.Management/StorageAccount/NewAzureStorageAccount.cs]: I have regenerated the help with the change.

File: src/Storage/Storage.Management/ChangeLog.md (hunk 1 above)
Comment (blueww) [reply src/Storage/Storage.Management/ChangeLog.md]: fixed.

**Comment by blueww** [comment]
@cormacpayne 
I have fixed both of you comments. (as the change is simple, amend it to origin commit. )
Please help to check and merge it.
Thanks!

**Pull request merged**